waitress/app/utils.py

```diff
@@ -113,7 +115,7 @@ def update(cls, trim=False):
             cls.user_queryset = SlackUser.objects.all()
             difference = cls.difference(members, cls.user_queryset)
 
-            if len(difference):
+            if len(difference) > 0:
```

Comment:
0 is a falsey value, in this case, perhaps living it as is was is okay for brevity

Reply:
corrected